GetRecd/Services/AuthService.swift

```diff
@@ -39,22 +39,26 @@ class AuthService: NSObject, GIDSignInDelegate {
                 print(error.localizedDescription)
                 return
             }
+            guard let user = user else {return}
             var userData = [String:Any]()
-            if (user!.displayName != nil) {
-                userData["name"] = user?.displayName!
+            if (user.displayName != nil) {
+                userData["name"] = user.displayName!
             }
-            if (user!.email != nil) {
-                userData["email"] = user?.email!
+            if (user.email != nil) {
+                userData["email"] = user.email!
             }
-            if (user!.photoURL != nil) {
-                userData["photoURL"] = user?.photoURL!.absoluteString
```

Comment:
Choose either ‘photoURL’ or ‘profilePictureURL’. Whichever you go with, change all instances of profile picture in the codebase to the correct term. Just do a global search for both and change them so they’re all the same. In the database it’s ‘photoURL’  currently so if you change to ‘profilePictureURL’, make sure to manually edit the Firebase entries to prevent crashing

Reply:
So, I changed it from `photoURL` to `profilePictureURL` because that is the name of the corresponding field in the `User` class. I wanted it to be a 1-1 correspondence. I also went through the Firebase, and I just deleted any entries that used the old model. This is okay because the new (and correct) database entries will simply be recreated when the user signs back in again.

Reply:
Change any other instances of ‘photoURL’ in the codebase if you haven’t already. I used ‘photoURL’ in the User class in my last PR since that’s what the DB had

Reply:
There are no other instances of `photoURL` in the codebase. If you have changed the name in `User`, then it hasn't been merged into the `sprint` branch.

Reply:
Changed.